src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -33,7 +33,7 @@ public class FunctionsSyncManager : IFunctionsSyncManager, IDisposable
         private const string TaskHubName = "taskHubName";
         private const string Connection = "connection";
         private const string DurableTaskV1StorageConnectionName = "azureStorageConnectionStringName";
-        private const string DurableTaskV2StorageOptions = "storageOptions";
+        private const string DurableTaskV2StorageOptions = "storageProvider";
```

Comment:
So it was incorrect from the initial commit? This code also exists in V2 of course.

Reply:
Yeah this was always incorrect, and I plan on porting this to the V2 branch as well.

In general, the durable extension uses storageProvider, so if someone is using this incorrect value for the sync triggers operation, their runtime wouldn't be respecting it anyways.